htsget-http-core/src/lib.rs

```diff
@@ -154,11 +154,11 @@ mod tests {
     request.insert("start".to_string(), "149".to_string());
     request.insert("end".to_string(), "200".to_string());
     let mut headers = HashMap::new();
-    headers.insert("Range".to_string(), "bytes=0-3366".to_string());
+    headers.insert("Range".to_string(), "bytes=0-3465".to_string());
```

Comment:
Interesting change... nulchar or extra return on the file being removed?

Reply:
I'm not entirely sure, but it is because of the way the file was compressed. When changing the GC info field I had to change it in the uncompressed version of the vcf file, and re-compress it to get a .vcf.gz file. Gzip also has a compression level between 1 and 9, which would impact the file size and regions.

I decided to make this change because it would allow the new tests that merge response tickets to work with noodles. And because the original sample1-bcbio-cancer.vcf file was incorrect - when looking at the records, the GC field is clearly not an Integer.